src/ProofOfConcepts/connectors/connectors.tsx

```diff
@@ -0,0 +1,363 @@
+import React from "react";
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from "@patternfly/react-core";
+import PlusCircleIcon from "@patternfly/react-icons/dist/esm/icons/plus-circle-icon";
+import OutlinedQuestionCircleIcon from "@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon";
+
+export const DataListSelectableRows: React.FunctionComponent = () => {
+  const [selectedDataListItemId, setSelectedDataListItemId] =
+    React.useState("");
+
+  const onSelectDataListItem = (id) => {
+    setSelectedDataListItemId(id);
+  };
+
+  const handleInputChange = (
+    id: string,
+    _event: React.FormEvent<HTMLInputElement>
+  ) => {
+    setSelectedDataListItemId(id);
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
Do we need this wrapping fragment?  If not it's worth removing it, even just to help reduce the indentation.